source/close.f90

```diff
@@ -28,6 +28,8 @@ subroutine closeUnits
 
   ! Then iterate through the first 1000 units
   do chkUnit=1, funit_minUnit-1
+    ! Do not close C/R files. handled by abend
+    if(chkUnit >= 91 .and. chkUnit <= 96) cycle
```

Comment:
Should probably also include `lout` and `6`. The comment should be "written to without preceding open in `abend`, closing handled in `abend`.

SixTrack version `v4.7.18` skips the following units in `closeUnits`: 5, 6, 36-58, 59-89 (STF), 91-97, 101-109, anything above 111 that's not 600 (if beam_expfile_open), not in DUMP, not dynksets.dat (aka 665). not a DYNKPIPE, not from SCATTER.

Reply:
Actually lout is written to in abend, but not closed. It probably should be closed.

Reply:
Most of these are either no loner in use, were never in use, or for the named files, have been replaced by dynamic units.

I'll add fort.6 to the list. Perhaps the track files too?